examples/rendy/prefab_data.rs

```diff
@@ -7,7 +7,7 @@ use amethyst::{
     utils::tag::Tag,
     Error,
 };
-use amethyst_rendy::{
+use amethyst::renderer::{
```

Comment:
Heya, `cargo fmt` groups it into the previous import, could you also commit that change?
Thanks!